api/models/Example.js

```diff
@@ -31,7 +31,21 @@ module.exports = {
 
 
   options: {
-    underscored: true
+    underscored: true,
+
+    classMethods: {
+      findPackageExamples: function(packageName, topic) {
+        return Example.findAll({
+          include:[
```

Comment:
Is this the patching that you talked about to speed up the query? Maybe useful to add a comment about it?